.github/CONTRIBUTING.md

```diff
@@ -146,7 +143,7 @@ Color names, values and descriptions are managed in Invision’s [Design System
 
 ### Changing a color’s name
 
-Color names are editable in <https://shopify.invisionapp.com/dsm/shopify/design-tokens> or via the Sketch DSM plugin.
+Color names are editable in <https://shopify.invisionapp.com/dsm/shopify/design-tokens> or via the Sketch Craft plugin, under the Design System Manager tab.
```

Comment:
I know if wasn't part of your diff but thoughts on making this being readable text that links to Invision rather than it being a bare url?